### PR TITLE
[Tests] Adding Color Loop set tests for Github CI

### PR DESCRIPTION
**Pull request by mkardous-silabs**
#### Description
Tests for Color Loop Set take too long to run on the Github CI. Adding a copy / paste of the certification tests but with shorter delays.

#### Tests
CI to validate that the test passes

### CODE REVIEW

File: src/app/tests/suites/TestColorControl_9_1.yaml [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+              - name: "Direction"
+                value: 0
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
+
+    - label: "Wait for 1000ms"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/TestColorControl_9_1.yaml:60]: The label here does not match the actual delay.

File: src/app/tests/suites/TestColorControl_9_1.yaml [hunk 2]
```diff
@@ -0,0 +1,770 @@
...
+              - name: "Direction"
+                value: 0
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
+
+    - label: "Wait 1000ms"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/TestColorControl_9_1.yaml:457]: Label does not match delay.

File: src/app/tests/suites/TestColorControl_9_2.yaml [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+              - name: "Direction"
+                value: 0
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
+
+    - label: "Wait for 1000ms"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/TestColorControl_9_2.yaml:63]: Does not match delay.

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/tests/suites/TestColorControl_9_1.yaml (hunk 1 above)
Comment (mkardous-silabs) [reply src/app/tests/suites/TestColorControl_9_1.yaml:60]: done

File: src/app/tests/suites/TestColorControl_9_1.yaml (hunk 2 above)
Comment (mkardous-silabs) [reply src/app/tests/suites/TestColorControl_9_1.yaml:457]: done

File: src/app/tests/suites/TestColorControl_9_1.yaml [hunk 4]
```diff
@@ -0,0 +1,770 @@
...
+                value: 160
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
+
+    - label: "Read ColorLoopStartEnhancedHue attribute from DUT"
+      command: "readAttribute"
+      attribute: "ColorLoopStartEnhancedHue"
+
```
[the remark this answers is not in the document]
Comment (mkardous-silabs) [reply src/app/tests/suites/TestColorControl_9_1.yaml:174]: done

**Review by jmartinez-silabs — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
@mkardous-silabs `TestColorControl_9_1` is failing on both Linux and Mac:
```
ERROR   19:09:08.883 - TEST OUT  : CHIP:TOO:  ***** Test Step 15 : Read ColorLoopStoredEnhancedHue attribute from DUT
...
ERROR   19:09:08.888 - TEST OUT  : CHIP:TOO:  ***** Test Failure: colorLoopStoredEnhancedHue value mismatch: expected 16384 but got 0
```

**Comment by mkardous-silabs** [comment]
> @mkardous-silabs `TestColorControl_9_1` is failing on both Linux and Mac:
> 
> ```
> ERROR   19:09:08.883 - TEST OUT  : CHIP:TOO:  ***** Test Step 15 : Read ColorLoopStoredEnhancedHue attribute from DUT
> ...
> ERROR   19:09:08.888 - TEST OUT  : CHIP:TOO:  ***** Test Failure: colorLoopStoredEnhancedHue value mismatch: expected 16384 but got 0
> ```

I'll check... I ran the test with an efr32 and it passed several times without any failures...

**Pull request merged**